src/LegacyApp.Api/Controllers/OrderController.cs

```diff
@@ -20,18 +22,13 @@ public OrderController(ILogger<OrderController> logger)
     [HttpPost(Name = "SubmitOrder")]
     public async Task<IActionResult> PostOrder(OrderModel order, [FromServices] IMessageSession session)
     {
-        var requestKey = Guid.NewGuid().ToString();
```

Comment:
This was purely showing/verifying that headers are returned by the RespondAsync method automatically. It can be any header, but it's a header.

Reply:
Apologies, I was under the assumption (from the video) that you didn't know why the header was there. I'll bring it back.